### PR TITLE
Feat/cse 432 breadcrumb links

### PR DESCRIPTION
**Pull request by bmikkelsen22**
# Changes introduced
- updated the url breadcrumb links pointed to
- cleaned up unnecessary [non pure](https://en.wikipedia.org/wiki/Pure_function ) call in breadcrumbs functional component
- load search results when navigating to search page with params or query string
- keyboard accessibility for breadcrumb dropdown and download modal

## Related issue(s):

- https://sbacjira.atlassian.net/browse/CSE-432
- https://sbacjira.atlassian.net/browse/CSE-426

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [x] Added new stories for created/modified components (if applicable)
- [x] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [ ] Checked out the branch and tested changes locally
- [ ] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

**Comment by bmikkelsen22** [comment]
@osu-cass/sb-reviewers This should be ready, but let me know if I need to do something about the duplicated code. They're just keyboard event handlers in separate components, so I figured they could stay.

File: src/components/BreadcrumbDropDown/index.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (tnoelcke) [inline src/components/BreadcrumbDropDown/index.spec.tsx:1]: Thanks for adding tests to cover the code you added!

File: src/components/BreadcrumbDropDown/index.tsx [hunk 2]
```diff
@@ -45,6 +45,12 @@ export class BreadcrumbDropDown extends Component<
     this.setState({ expanded });
   };
 
+  handleEnterPress = (e: React.KeyboardEvent<HTMLLIElement>) => {
+    if (e.key === 'Enter') {
+      this.onClick();
+    }
+  };
```
Comment (tnoelcke) [inline src/components/BreadcrumbDropDown/index.tsx:52]: I like this. This is a nice simple solution. Nice work.

**Review by tnoelcke — APPROVED** [review]
Looks good. Nice work.

**Pull request merged**